### DISCUSSION TITLE
Java 17 support

### DISCUSSION THREAD

**Discussion by eshankiyer** [FabricMC/fabric-api#1744] [Player support]
Java 17 GA has just released, and I tried it out with fabric. It didn't work but I was just wondering what would be the status on implementing it

**Comment by modmuss50** [comment]
It should work, I have been using Java 17 with fabric for a while. Do you get a crash log?

**Comment by eshankiyer** [comment]
Hmmmm, it seems to work when I remove Hydrogen, probably because it makes calls to java 16 specific apis, but I'm not sure
